src/pages/api/admin/updateTopicInfo.ts

```diff
@@ -0,0 +1,28 @@
+import { ClassModel } from '@server/db/class'
+import { apiController } from '@utils/api'
+import { Types } from 'mongoose'
+
+export interface UpdateTopicInfoParams {
+  cid: string
+  topics: string[]
+}
+
+export interface UpdateTopicInfoResults {
+  topics: string[]
+  success: boolean
+}
+
+export default apiController<UpdateTopicInfoParams, UpdateTopicInfoResults>(async ({ cid, topics }) => {
+  const CourseClass = await ClassModel.findById(new Types.ObjectId(cid))
```

Comment:
PascalCase is used for Class names. We are using camelCase for variable names.

```suggestion
  const courseClass = await ClassModel.findById(new Types.ObjectId(cid))
```

Reply:
> PascalCase is used for Class names. We are using camelCase for variable names.

updated it here https://github.com/kixlab/kuiz/pull/46/commits/2e04c862f97ba3b918c13396232055102a87d746